openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -567,7 +568,7 @@ class DeactivateLogoutView(APIView):
     -  Log the user out
     - Create a row in the retirement table for that user
     """
-    authentication_classes = (JwtAuthentication, SessionAuthentication,)
+    authentication_classes = (JwtAuthentication, SessionAuthentication, BearerAuthentication)
```

Comment:
```suggestion
    # BearerAuthentication is added here to support account deletion from the mobile app until it moves to JWT Auth.
    authentication_classes = (JwtAuthentication, SessionAuthentication, BearerAuthentication)
```

Reply:
@feanil @oksana-slu this issue includes adding JWT support to new mobile apps https://github.com/openedx/edx-platform/issues/33307

Reply:
Resolved in this commit: 6a1f126be917c69aaa6d0992b754701d701c09ff

Reply:
It looks like that commit is not on this PR yet but assuming it will be eventually.

Reply:
Just GitHub link problem, this commit can be seen via this link because it's committed to a fork: [pull/33289/commits/6a1f126be917c69aaa6d0992b754701d701c09ff](https://github.com/openedx/edx-platform/pull/33289/commits/6a1f126be917c69aaa6d0992b754701d701c09ff)